develop/potions.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Potions
+description: Learn how to add a custom potion for various status effects.
+authors:
+  - dicedpixels
+  - PandoricaVi
+  - Drakonkinst
+---
+
+# Potions
+
+Potions are consumables that grants an entity an effect. A player can brew potions using a Brewing Stand or obtain them
+as items through various other game mechanics.
+
+## Custom Potions
+
+Adding a potion follows a similar path as adding an item. You will create an instance of your potion and register it by
+calling `BrewingRecipeRegistry.registerPotionRecipe`.
+
+::: info
+When Fabric API is present, `BrewingRecipeRegistry.registerPotionRecipe` is made accessible through an Access Widener.
+:::
+
+### Creating the Potion
+
+Let's start by declaring a field to store your `Potion` instance. We will be directly using the initializer class to
+hold this.
+
+@[code lang=java transclude={20-28}](@/reference/latest/src/main/java/com/example/docs/potion/FabricDocsReferencePotions.java)
```

Comment:
Seems to be the wrong transclude line numbers?

![image](https://github.com/FabricMC/fabric-docs/assets/93472213/34d483f0-10cb-4b04-86b1-ade116ce3ba0)